modules/core/src/lib/tooltip.ts

```diff
@@ -62,6 +62,9 @@ export default class Tooltip {
       return;
     }
 
+    el.style.display = 'block';
+    el.style.transform = `translate(${x}px, ${y}px)`;
```

Comment:
There's no point setting `transform` if the tooltip is going to be hidden.
I would suggest keeping this block in its original place, instead move `Object.assign(el.style, displayInfo.style)` to after these lines.

Reply:
I can do that but then once a tooltip is provided style `display: none` by the user it'll persist and no tooltips will be shown afterwards (even when no display style is provided).

I'll update my codepen (in the previous comment) to show this.

Reply:
Done